README.md

```diff
@@ -30,17 +30,19 @@ things you'll need to do first.
     to add these credentials to your keychain, say yes. *You will not be able*
     *to see your password*
 
-## Deploying to Heroku
+## Deploying to Heroku: Checklist
 
 Now you're set up to use Heroku. To deploy a new application
 to Heroku:
 
 -   [ ] Run `heroku create` in the command line in the root of your Rails API to
     create a new (blank) app on Heroku.
 -   [ ] Push your latest code to Heroku (`git push heroku master`)
--   [ ] Tell Heroku to run your migration files (`heroku run rake db:migrate`).
+-   [ ] Update your heroku database by telling Heroku to run your migration files (`heroku run rake db:migrate`).
     If you have any other rake tasks that need to run (e.g. `rake db:seed`), run
     those with `heroku run` as well.
+-   [ ] Set your secrets using Heroku run.
```

Comment:
So I just added that to be part of a general check list. Originally that section read like a checklist but skipped some parts (namely setting secrets for your heroku app) and that's a step I saw some people skipping in 015. 

Pushed changes in 0c4da27270cc75d3c76c14fc84506a859a4bb5d6 Is that better?